### PR TITLE
{bio}[intel/2016b] GRIT v2.0.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GRIT/GRIT-2.0.5-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ('pysam', '0.9.1.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysam/'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/nboley/grit/archive/'],
+        'source_tmpl': '%(version)s.tar.gz',
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/GRIT/GRIT-2.0.5-intel-2016b-Python-2.7.12.eb]: not a block for this, but shouldn't this happen automatically when the extension is `PythonPackage`?

**Comment by wpoely86** [comment]
:+1:

File: easybuild/easyconfigs/g/GRIT/GRIT-2.0.5-intel-2016b-Python-2.7.12.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/g/GRIT/GRIT-2.0.5-intel-2016b-Python-2.7.12.eb]: open an issue for this in the easyblocks repo, I'll see if we can make that happen (or why it doesn't happen already)

I think the problem is that we don't want *every* extension to prepend to `$PYTHONPATH`, it just needs to be done once...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2636.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f73515233a4a346d347d69e793782485 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0f47a5cf4f64b91ba7df40f5e7ddeb56 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**